doc/api/http.md

```diff
@@ -1406,6 +1406,20 @@ This event is guaranteed to be passed an instance of the {net.Socket} class,
 a subclass of {stream.Duplex}, unless the user specifies a socket
 type other than {net.Socket}.
 
+### Event: `'dropRequest'`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `request` {http.IncomingMessage} Arguments for the HTTP request, as it is in
+  the [`'request'`][] event
+* `socket` {stream.Duplex} Network socket between the server and client
+
+When the number of requests on a socket reaches the threshold of
+`server.maxRequestsPerSocket`, the server will drop new requests
+and emit `'dropRequest'` event instead.
```

Comment:
I would specify that the server writes a 503 response after the event is emitted.